src/servers/common.cc

```diff
@@ -316,6 +316,35 @@ GetDataTypeByteSize(const std::string& protocol_dtype)
   }
 }
 
+size_t
```

Comment:
We should have this code duplication. Did you prepend the call with the namespace? Namespace "ni" should already be defined in http_server.cc, so ni::GetDataTypeByteSize should work.

Reply:
Fixed. Removed code duplication